common/scripts/lib/_helpers.sh

```diff
@@ -325,7 +325,7 @@ __set_db_ip() {
   __process_success "Please enter the IP address of the database or D to set the default ($db_ip)."
   read response
 
-  if [ "$response" != "D" ]; then
+  if [ "$response" != "D" ] && ["$response" != "d"]; then
```

Comment:
why is this not being converted to toUpper while other inputs are ?

Reply:
@ric03uec can we give non - ip inputs for admiral IP? those urls will be converted to uppercase. 
also if i give inputs like `localhost` it gets converted to upper case.